python/Ganga/GPIDev/Lib/Job/Job.py

```diff
@@ -671,13 +625,19 @@ def postprocessoutput(self, outputfiles, outputdir):
                     for currentFile in glob.glob(os.path.join(outputdir, outputfile.namePattern)):
                         os.system("gzip %s" % currentFile)
 
+
             backend_output_postprocess = self.getBackendOutputPostprocessDict()
             if backendClass in backend_output_postprocess:
                 if outputfileClass in backend_output_postprocess[backendClass]:
-                    if backend_output_postprocess[backendClass][outputfileClass] == 'client':
+                    if backend_output_postprocess[backendClass][outputfileClass].lower() == 'client':
                         logger.info("Job %s Putting File %s: %s" % (self.getFQID('.'), getName(outputfile), outputfile.namePattern))
                         outputfile.put()
                         for f in glob.glob(os.path.join(self.outputdir, outputfile.namePattern)):
+                            # FIXME should this be controlled via some per filetype config?
```

Comment:
No, it should most likely be solved using polymorphism rather than an explicit type check.